source/blood/src/view.cpp

```diff
@@ -2027,7 +2031,21 @@ void viewResizeView(int size)
         gViewY1S = divscale16(gViewY1, yscale);
     }
     videoSetViewableArea(gViewX0, gViewY0, gViewX1, gViewY1);
-    gGameMessageMgr.SetCoordinates(gViewX0S + 1, gViewY0S + 1);
+    if (gViewMode == 4) // 2D map view
+    {
+        int nOffset = divscale16(bDrawFragsBg ? (tilesiz[2229].y*ydim*((gNetPlayers+3)/4))/200 : 0, yscale);
```

Comment:
something like this might be better
`int nOffset = bDrawFragsBg ? tilesiz[2229].y*((gNetPlayers+3)/4) : 0;`